pyupgrade.py

```diff
@@ -2029,6 +2038,19 @@ def _replace_yield(tokens: List[Token], i: int) -> None:
     tokens[i:block.end] = [Token('CODE', f'yield from {container}\n')]
 
 
+def _replace_six_calls(
+    tokens: List[Token], call: ast.Call, i: int,
+    needs_parenthesis: bool = False,
```

Comment:
let's make this keyword-only so it isn't a [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap)

Reply:
Uhm, are you sure ? Needs default values (None) -> needs `Optional` typing, needs None checks ... looks, well, non-optimal

Reply:
yes I'm sure, but you're not understanding me correctly -- check out this video https://youtu.be/aKCfCmSggPg?list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY

Reply:
Oh - that one, got it.